packages/SystemsThinkingWorkbench-Core.package/STWNode.class/instance/createConnectionFrom.ofType..st

```diff
@@ -0,0 +1,10 @@
+interaction
+createConnectionFrom: startingNode ofType: aSymbol
+
+	| connection classType |
+	
+	(aSymbol == #transfer) ifTrue: [classType := STWTransferConnection].
```

Comment:
I would like to get a third opinion on whether this should be a symbol or a class passed as type parameter.

I would prefer a class, it would not required updating this when adding new connections (and also external devs could add their own connections).